command.go

```diff
@@ -37,7 +36,7 @@ type Command struct {
 	// Execute this function if a usage error occurs.
 	OnUsageError OnUsageErrorFunc
 	// List of child commands
-	Subcommands Commands
```

Comment:
Was there a reason you dropped this? I kind of like the opaque type in case we want to hang any other behavior on it.